src/core/libmaven/mzUtils.cpp

```diff
@@ -6,6 +6,24 @@
  */
 namespace mzUtils {
 
+    string replaceAll(string thisOne, string fromThis, string withThis){
```

Comment:
why not use the [standard function](http://en.cppreference.com/w/cpp/algorithm/replace) ?